lib/cudadrv/memory.jl

```diff
@@ -759,6 +759,8 @@ memory_type(x) = CUmemorytype(attribute(Cuint, x, POINTER_ATTRIBUTE_MEMORY_TYPE)
 
 is_managed(x) = convert(Bool, attribute(Cuint, x, POINTER_ATTRIBUTE_IS_MANAGED))
 
+host_device(x) = convert(Int, attribute(Cuint, x, POINTER_ATTRIBUTE_DEVICE_ORDINAL))
```

Comment:
I would call it `CuDevice`, and also return a `CuDevice`. That way it is consistent with e.g. `CuDevice(::CuContext)`.

Reply:
Great idea. Updated code + test.